src/components/layout/MobileNavbar.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { graphql, StaticQuery, Link } from 'gatsby';
+import styled from 'styled-components';
+import { CSSTransition } from 'react-transition-group';
+
+class MobileNavBar extends React.Component {
+  state = {
+    active: null,
+    showSubMenu: false,
+  };
+
+  handleMenuClick = clicked => {
+    clicked === this.state.active
+      ? this.setState({ showSubMenu: false })
+      : this.setState({ active: clicked, showSubMenu: true });
+  };
+
+  renderMenuItems = pages =>
+    pages.map(menuItem => {
+      if (menuItem.node.parentPage === null) {
+        return (
+          <ParentMenuItem
+            key={menuItem.node.title}
+            onClick={() => this.handleMenuClick(menuItem.node.title)}
+            active={this.state.active === menuItem.node.title}
+          >
+            {menuItem.node.title}
+          </ParentMenuItem>
+        );
+      }
+      return null;
```

Comment:
The effect might be the same, but I think it would look nicer if you would filter the pages list on top level pages before passing it to this function instead of returning null here.

Reply:
Fixed in #94379c0